openapi.json

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
You may use https://github.com/Akkudoktor-EOS/EOS/blob/main/docs/akkudoktoreosserver/openapi.json for testing. This way the documentation will stay in sync with the server api.